katello/katello-debug.sh

```diff
@@ -125,5 +125,7 @@ echo "db.task_status.find({state:{\$ne: \"finished\"}}).pretty().shellPrint()" >
 
 add_cmd "mongo pulp_database /tmp/pulp_running_tasks.js" "pulp-running_tasks"
 
+add_cmd "hammer ping" "hammer-ping"
```

Comment:
@lzap it should be ok here is the output with the commit in it:

[root@centos7-katello-3-3 foreman-debug-6Jnku]# cat hammer-ping 
COMMAND> hammer ping

candlepin:      
    Status:          ok
    Server Response: Duration: 32ms
candlepin_auth: 
    Status:          ok
    Server Response: Duration: 27ms
pulp:           
    Status:          ok
    Server Response: Duration: 152ms
foreman_tasks:  
    Status:          ok
    Server Response: Duration: 45ms